src/components/NavBar.js

```diff
@@ -1,31 +1,63 @@
-import React from "react";
-import {BrowserRouter as Router, Link} from "react-router-dom";
+import React, {useState} from "react";
+import {Link} from "react-router-dom";
 import { faCaretDown } from "@fortawesome/free-solid-svg-icons";
+import { faQuestionCircle } from "@fortawesome/free-solid-svg-icons";
+import { faHammer } from "@fortawesome/free-solid-svg-icons";
+import { faTools } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
 import './NavBar.css';
 import OfekLogo from '../assets/images/OfekLogo.png';
 
-const NavBar = () => {
+const NavBar = props => {
```

Comment:
Destructuring props like that:
```const NavBar = ({MinMaxMain ,mini})=> {...```

than use them neutral in the code

Reply:
one of the props is a function so I declared props types above

Reply:
Even if one or more of your props is a function, you can still destrcut your props (which I highly suggest you to do)